### PR TITLE
Header and footer features PR#23

### PR DESCRIPTION
**Pull request by efmomoh**
### What does this PULL REQUEST do?

**@Obote  and @efmomoh , we:**

- [ ] Selected topic of choice.
- [ ] Create a page header according to the wireframe
- [ ] Create a footer according to the wireframe

### CODE REVIEW

**Comment by Obote** [comment]
Hi @efmomoh , 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... to merge :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All tasks have been fairly assigned ✔️ ✔️ 
- Your header has been properly set up 👏 👏 
- Your footer has been properly set up 👏 👏 


Cheers and Happy coding!👏👏👏

File: src/style.css [hunk 1]
```diff
@@ -22,18 +24,100 @@
...
     width: 100%;
     height: auto;
+    box-sizing: border-box;
   }
 
   .h1 {
     font-size: 2.5rem;
     font-weight: 700;
     font-family: serif;
     text-align: center;
```
Comment (Obote) [inline src/style.css:39]: It's nice you have done using media queries, it would be nice if avoided repetition.

**Review by Obote — APPROVED** [review]
Hi @efmomoh , 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... It's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All tasks have been fairly assigned ✔️ ✔️ 
- Your header and footer has been properly set up 👏 👏 

## Optional suggestions

Please check the comments section for suggestion 

Cheers, and Happy coding!👏👏👏

File: src/style.css (hunk 1 above)
Comment (efmomoh) [reply src/style.css:39]: Ok, thank you for the feedback. I will address it but I think the media query can use the same classes at the same time different classes if one chooses. However, I will look at it. Thank you again.

**Pull request merged**